### DISCUSSION TITLE
Opening items from notifications doesn't seem to mark them as read

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#46274] [Mobile] [answered]
GitHub Mobile beta

1. Open the notifications view (which has a filter for unread)
![Screenshot_20230203-084019.png](https://user-images.githubusercontent.com/2119212/216617728-f1faf758-9b18-4739-91cb-2c7e0321c2b7.png)
2. Tap an item
3. Go back to the notifications view
4. Tap another item
5. Repeat 3&4 for most items
6. Note that the view doesn't update. 

Expected results, returning to the view should update the list.

Here's Chrome for Android
![Screenshot_20230203-084352.png](https://user-images.githubusercontent.com/2119212/216618270-f8cfacb2-d136-4c17-bd7c-74b02ae515f0.png)

Note that dragging down from the top of the screen would probably fix this, but I shouldn't have to do that. And it isn't just using the back button, even switching to the view by tapping the button at the bottom of the window doesn't tend to update it.

**Comment by mattruggio** [comment, marked as answer]
Hey @jsoref, thanks for the heads up on this!  This makes sense and looks to be a bug.  I will file a user experience issue with our mobile team for further investigation.
